layouts/partials/posts/featured.html

```diff
@@ -7,7 +7,11 @@
         <span class="date">{{ .Date.Format (.Site.Params.DateFormat | default "January 2, 2006") }}</span>
         {{ end }}
         <h2><a href="{{ .Permalink }}">{{ .Title }}</a></h2>
+        {{ if .Description }}
         <p>{{ .Description }}</p>
+        {{ else }}
```

Comment:
Could this be changed into an `else if` in case there is no `Summary` either?

Reply:
See point 2 in my answer below.